ci-operator/step-registry/openshift/e2e/libvirt/test/openshift-e2e-libvirt-test-commands.sh

```diff
@@ -326,6 +326,12 @@ export KUBE_TEST_REPO_LIST=${SHARED_DIR}/kube-test-repo-list
         TEST_ARGS="${TEST_ARGS:-} --from-repository=quay.io/multi-arch/community-e2e-images"
         ;;
     esac
+	
+	case "${CLUSTER_TYPE}" in
+	powervs*)
+		TEST_ARGS="${TEST_ARGS:-} --disable-monitor=service-type-load-balancer-availability"
+    	;;
+	esac
```

Comment:
Cool!

Reply:
Thanks Paul, Shall we get this PR merged to skip the tests for now until the issue is fixed?

Reply:
Yes, I completely agree